dask/dataframe/core.py

```diff
@@ -4544,7 +4544,7 @@ def merge(
             broadcast=broadcast,
         )
 
-    @derived_from(pd.DataFrame)  # doctest: +SKIP
```

Comment:
The skip was added in https://github.com/dask/dask/pull/6397/commits/bf270a4f47d6ea36ab3d4a9f9a194ebbd4286d29. The relevant context in the underlying PR appears to center around this comment https://github.com/dask/dask/pull/6397#issuecomment-706435632.

I suspect we no longer need this skip now but will defer to you, since you helped review that PR and probably has more context re its original purpose.

Reply:
Thanks for the pointer! So based on that it seems like if the tests pass on CI we are good to go.